Sources/MapboxSearch/PublicAPI/Use Cases/Place Autocomplete/PlaceAutocomplete.swift

```diff
@@ -185,51 +270,47 @@ private extension PlaceAutocomplete {
         }
         
         let dispatchGroup = DispatchGroup()
+        var resolvingError: Error?
         
-        var resolvedResultsUnsafe: [SearchResult?] = Array(repeating: nil, count: suggestions.count)
+        var resolvedSuggestions: [Suggestion?] = Array(repeating: nil, count: suggestions.count)
         let lock = NSLock()
 
         suggestions.enumerated().forEach { iterator in
```

Comment:
There were differences between initial iOS and Android implementations. iOS didn't use batch resolving, so I kept this logic despite the difference with Android.
Because I don't have all context and don't know the reasons that initially lead to avoiding batch resolve

Reply:
It's ok, batch resolve would be just quicker, but suggestions can be resolved one by one as well.